### DISCUSSION TITLE
cominication with px4 SITL and gazebo and mrs uav system

### DISCUSSION THREAD

**Discussion by mrshab72** [ctu-mrs/mrs_uav_system#73] [Q&A]
hi every body
I use the [px4 SITL and gazebo](https://docs.px4.io/master/en/simulation/gazebo.html)  simulation for quadrotor  and now want to add mrs packages to this simulation for create a multi vehicle scenario.
can you introduce a specific process to do this work?
thanks alot

**Comment by matemat13** [comment]
Hello, take a look at our simulation package here: https://github.com/ctu-mrs/simulation/
You can use this tmux script to launch a simulation with multiple drones: https://github.com/ctu-mrs/simulation/tree/master/example_tmux_scripts/many_drones_gps
